### PR TITLE
Added OpInfo tests for NNC

### PR DESCRIPTION
**Pull request by Chillee**
Finds a couple of bugs:

1. permute needs to wrap dimensions
2. slice needs to wrap dimensions
3. frac doesn't work correctly for negative values
4. Permute has some other failures.

This PR also fixes 1 + 2.

### CODE REVIEW

File: test/test_jit_fuser_te.py [hunk 1]
```diff
@@ -1803,5 +1807,130 @@ def eager(x, y, z: int):
...
+            self.assertTrue(op.name not in work_list)
+            if "Unhandled node kind" in str(e):
+                print(e)
+                return
+            if "UNSUPPORTED DTYPE" in str(e):
+                print(e)
+                return
+            if "UNSUPPORTED DTYPE" in str(e):
+                print(e)
+                return
```
Comment (ZolotukhinM) [inline test/test_jit_fuser_te.py:1927]: Should we rethrow the exception in this case? Also, the dtype check is duplicated

**Review by ZolotukhinM — APPROVED** [review]
LGTM!

> This PR also fixes 1 + 2.

Cool that you fixed 3! :)

File: test/test_jit_fuser_te.py (hunk 1 above)
Comment (Chillee) [reply test/test_jit_fuser_te.py:1927]: Well, if we rethrow the exception the test will fail :thinking:

Comment (ZolotukhinM) [reply test/test_jit_fuser_te.py:1927]: I mean, fail is the correct behavior here :) Could you just comment out the ops that are failing from the list and put a TODO to fix them?

Comment (Chillee) [reply test/test_jit_fuser_te.py:1927]: The reason it's written this way is that the vast majority of ops are actually not supported (i.e. they don't have lowerings). So, the ops I have in here are a whitelist.

I wouldn't be too opposed to having a very large blacklist, but one issue I've found with OpInfo is that it's an open database - that is, other people will add tests. So, keeping a blacklist causes a maintenance burden whenever people add new tests that fail.

If I could, I would prefer something like "expectedFailure", but I don't think there's any way of doing that here.

Comment (ZolotukhinM) [reply test/test_jit_fuser_te.py]: So, from the ops from the work_list there are some that throw exceptions here, right? Can we comment them out from the work_list until we fix them?

Comment (Chillee) [reply test/test_jit_fuser_te.py]: No there aren't - the "work_list" is only ops that work without exceptions.

**Pull request closed without merging**